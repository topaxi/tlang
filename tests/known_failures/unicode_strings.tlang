// Tests unicode and special string handling
// May have different behavior between interpreter and JavaScript

fn unicode_test() {
    let unicode_str = "Hello 🌍";
    unicode_str |> log();
    
    // Test string length with unicode
    let length_test = "café";
    length_test |> log();
}

fn special_chars_test() {
    let newline_str = "line1
line2";
    newline_str |> log();
    
    let tab_str = "col1	col2";
    tab_str |> log();
    
    let simple_str = "He said hello";
    simple_str |> log();
}

log("=== Unicode and Special Characters Test ===");
unicode_test();
special_chars_test();