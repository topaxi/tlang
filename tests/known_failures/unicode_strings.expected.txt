=== Unicode and Special Characters Test ===
Hello 🌍
café
line1
line2
col1	col2
He said hello